sklearn/linear_model/coordinate_descent.py

```diff
@@ -1255,7 +1269,10 @@ class LassoCV(LinearModelCV, RegressorMixin):
         (e.g. data is expected to be already centered).
 
     normalize : boolean, optional, default False
-        If ``True``, the regressors X will be normalized before regression.
+        If True, the regressors X will be normalized before regression.
+        Normalization makes the `coef_` independent from the number of training
+        samples. If you wish to standardize instead, please use
```

Comment:
I find the sentence "Normalization makes the `coef_` independent from the number of training samples." ambiguous.